packages/libs/react-ui/src/components/Grid/Grid.stories.tsx

```diff
@@ -28,14 +28,14 @@ type Story = StoryObj<typeof Grid.Container>;
 export const Primary: Story = {
   name: 'Grid',
   args: {
-    spacing: 'xl',
+    spacing: '$xl',
   },
   render: ({ spacing }) => (
     <>
       <Grid.Container spacing={spacing}>
-        {Array.from(new Array(12)).map((i) => (
+        {Array.from(new Array(12)).map((empty, i) => (
```

Comment:
can you use _ in the place of empty? Might look a bit cleaner

Reply:
I agree! sadly, it raises a warning with the linter... should we use `_` and add an ignore line?